README.md

```diff
@@ -22,7 +22,7 @@ functional btest setup. You can add the following optional features:
   (where you'll define log streams, handle runtime events, etc)
   directly in the toplevel `scripts` folder.
 
-- `spicy-analyzer` adds a Spicy analyzer to the package. The feature relies on
+- **TODO: Update** `spicy-protocol-analyzer` adds a Spicy protocol analyzer to the package. The feature relies on
```

Comment:
The TODO here is to cover the three Spicy analyzer types, right?

Reply:
Oh, missed updating that, will do.